src/app/builders/components/Table/BuildersTable.tsx

```diff
@@ -245,9 +251,11 @@ export const BuildersTable = ({ filterOption }: { filterOption: BuilderFilterOpt
           </thead>
           <Suspense fallback={<div>Loading table data...</div>}>
             <tbody>
-              {rows.map(row => (
-                <BuilderDataRow key={row.id} row={row} />
-              ))}
+              {rows.map((row, index) => {
```

Comment:
NIT: `index` isn't used
```suggestion
              {rows.map((row) => {
```